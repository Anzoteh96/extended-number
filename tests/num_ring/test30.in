-16925761244760639663365728295831936095554633677751079835588872715448096844220630433872232
512212933183202937506327848642853769160423689076337281455021592807474790009868530112509584
301827672849994213927714186812400383124397420486227317646256862655587700588275644505721061
