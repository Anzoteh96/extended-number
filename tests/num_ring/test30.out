Enter the number base b:
Enter two numbers n and m:
b is 16925761244760639663365728295831936095554633677751079835588872715448096844220630433872232
n mod b is 4440095840383747605355999767895686293784678743804886387355411344031884683249617096342624
m mod b is 14089731689063339650496805783257469499968647964458960441246026492970054236524927129893117
n + m is 1604066284686447592487077255321219698198693030512766993012565121553842075553913792363509
n - m is 7276125396081047618224922280470152889370664457097005781698257566509927290945320400321739
n * m is 6198273790663842892200630304983863265987423851935801784028256843549699883011870672581056
n ** m is 319369399150585121081389341596877370558087704444102646733766477410626087069615199869120
